test(ivy): skip useJit tests with Ivy

crisbeto opened the pull request:
Currently the `useJit` option from `TestBed.configureCompiler` isn't supported. These changes rework the existing test suites not to pass in `useJit` when running with Ivy.

These changes reduce the tests from the `test-fixme-ivy-aot //packages/core/test:test` target from `1264 specs, 515 failures` to `1037 specs, 306 failures`.

crisbeto commented on packages/core/test/linker/entry_components_integration_spec.ts:
```diff
@@ -6,16 +6,20 @@
-  fixmeIvy('unknown') && describe('no jit', () => { declareTests({useJit: false}); });
+  if (ivyEnabled) {
+    fixmeIvy('unknown') && describe('no jit', () => { declareTests(); });
+  } else {
```
Some doubts I had on these changes:

1. Should the `describe` say continue to say `no jit` like it does now or perhaps it should be changed to something like `describe('ivy', ...)`?
2. I'm not completely sure whether this is the proper way to gate these tests. Maybe there should be another `fixme` instead?

mhevery replied:
Yes I think we should change `no jit` to `ivy` LGTM otherwise.

mary-poppins commented:
You can preview baaccd7 at https://pr27067-baaccd7.ngbuilds.io/.

mhevery reviewed (approved):
Yes I think we should change `no jit` to `ivy` LGTM otherwise.

crisbeto committed:
test(ivy): skip useJit tests with Ivy

Currently the `useJit` option from `TestBed.configureCompiler` isn't supported. These changes rework the existing test suites not to pass in `useJit` when running with Ivy.

mary-poppins commented:
You can preview 8b2faac at https://pr27067-8b2faac.ngbuilds.io/.
